tagging/routes/routeGetMostUsedTags.go

```diff
@@ -54,3 +55,37 @@ func routeGetMostUsedTags(r *http.Request, a routes.Arguments) (int, interface{}
 		"mostUsedTags": tagsList,
 	}
 }
+
+func routeGetMostUsedTagsHistory(r *http.Request, a routes.Arguments) (int, interface{}) {
+	logger := jsonlog.LoggerFromContextOrDefault(r.Context())
+	u := a[users.AuthenticatedUser].(users.User)
+	tx := a[db.Transaction].(*sql.Tx)
+	dbRes, err := models.MostUsedTagsHistoryByUser(tx, u.Id)
+	if err != nil {
+		logger.Error("Could not fetch most used tags.", err.Error())
+		return http.StatusInternalServerError, nil
```

Comment:
From what I can see in other code, you can also return the details of the error as the second return value is an `interface{}` and something like `error` will be handled perfectly well, as can be seen from code in `users/create.go`, for example.